airflow/www/utils.py

```diff
@@ -68,17 +69,11 @@ def should_hide_value_for_key(key_name):
 
 
 def get_params(**kwargs):
-    params = []
-    for k, v in kwargs.items():
-        if k == 'showPaused':
-            # True is default or None
-            if v or v is None:
-                continue
-            params.append('{}={}'.format(k, v))
-        elif v:
-            params.append('{}={}'.format(k, v))
-    params = sorted(params, key=lambda x: x.split('=')[0])
-    return '&'.join(params)
+    if 'showPaused' in kwargs:
+        v = kwargs['showPaused']
+        if v or v is None:
+            kwargs.pop('showPaused')
+    return urlencode(kwargs)
```

Comment:
Hi @ashb 
There is a difference in the result. Let's say `kwargs` is `{'a':"100", "c":None}`:
Result from previous implementation: `'a=100'`
Result from your new implementation: `'a=100&c=None'`

(Actually there is a potential bug in the previous implementation, let's say my kwargs is `{'a':"100", "c":0}`. I would except `'a=100&c=0'` while it will return `'a=100'`)

Coming back to your change, I would suggest to add one more check to check if value is `None`: if yes, pop it out as well.

Reply:
Good call. Updated so that given `a=100, c=None` it will output `a=100&c=` (given `c` was passed as an arg I feel it should appear in the output string. This does mean it will come back in when parsed as `c: ""` but I feel that is more correct)

This function is only currently used from within generate_pages so we don't have to worry about this possibly slight change in behaviour